src/librustc_resolve/diagnostics.rs

```diff
@@ -26,7 +26,10 @@ use crate::{
     BindingError, CrateLint, HasGenericParams, MacroRulesScope, Module, ModuleOrUniformRoot,
 };
 use crate::{NameBinding, NameBindingKind, PrivacyError, VisResolutionError};
-use crate::{ParentScope, PathResult, ResolutionError, Resolver, Scope, ScopeSet, Segment};
+use crate::{
+    ParentScope, PathResult, ResolutionError, Resolver, Scope, ScopeSet, Segment,
+    UndeclaredLabelCandidate,
+};
```

Comment:
Style nit: could you avoid multi-line imports?

Reply:
Fixed!